MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -1459,6 +1463,40 @@ private void stzcpy()
             Registers.SetPointer(destinationPointer);
         }
 
+        /// <summary>
+        ///     Concatenates Two Strings with a fixed length
+        ///
+        ///     Signature: char *stzcat(char *dst,char *src,int num);
+        /// </summary>
+        private void stzcat()
+        {
+            var destinationPointer = GetParameterPointer(0);
+            var sourcePointer = GetParameterPointer(2);
+            var limit = GetParameter(4);
+
+            var stringLength = Module.Memory.GetString(destinationPointer, true).Length;
+
+            //Truncate Destination if LIMIT is less than DST 
+            if (stringLength > limit)
```

Comment:
would this ever happen, the string being longer than the total buffer length? That's a pretty major module bug hah

Reply:
I mean... it shouldn't....but it could.